aip/general/0213/aip.md

```diff
@@ -0,0 +1,214 @@
+# Common components
+
+In general, APIs should be designed to be self-contained. APIs generally need
+to be able to move forward independently of one another, and mutual
+dependencies can cause downstream APIs to be forced into taking major version
+changes or even lead to dependency conflicts.
+
+However, there are also cases where common structures are valuable, especially
```

Comment:
I think this isn't clear enough on when common components should be used vs not. Can a section be added to explain when it's appropriate?

Reply:
FWIW, this is more or less verbatim from google.aip.dev/213.  But I'll take a shot at clarifying.